### ISSUE TITLE
"Part" in filename causes introspection of day failure

### ISSUE THREAD

**Issue by Indivicivet** [wimglenn/advent-of-code-data#152]
./AoC2024/day14.py works fine, but none of ./AoC2024/day14part2.py ./AoC2024/day14_part2.py ./AoC2024/day14p2.py work.

```Traceback (most recent call last):
  File "D:\repos\AoC2024\day14_p2.py", line 1, in <module>
    from aocd import data
  File "C:\Program Files\Python312\Lib\site-packages\aocd\__init__.py", line 48, in __getattr__
    day, year = get_day_and_year()
                ^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python312\Lib\site-packages\aocd\get.py", line 201, in get_day_and_year
    raise giveup("Failed introspection of day")
aocd.exceptions.AocdError: Failed introspection of day
```

it'd be great if this was supported for people who want to keep part 1 and part 2 separate and clearly labelled, either because the approaches are different or because they're only doing one part in python.

**Comment by Indivicivet** [comment]
btw, if anyone else is coming here with the same request, I'm using `day14_.py` as a temporary workaround.

**Comment by wimglenn** [comment]
May I suggest "part A" and "part B" or some variant that? This would be in line with the existing aocd [submit](https://github.com/wimglenn/advent-of-code-data?tab=readme-ov-file#automated-submission) APIs ([`Literal["a", "b"]`](https://github.com/wimglenn/advent-of-code-data/blob/v2.0.4/aocd/types.py#L11)).

**Comment by Indivicivet** [comment]
thanks for the implementation and the suggestion!

**Comment by wimglenn** [comment]
This is released in v2.1.0.
